### PR TITLE
add the git hash to the settings struct

### PR DESCRIPTION
**Pull request by VDB-Bram**
see issue https://github.com/KULeuvenNeuromechanics/PredSim/issues/67

### CODE REVIEW

**Comment by Lars-DHondt-KUL** [comment]
looks good, but please merge to master-dev

File: DefaultSettings/get_git_hash.m
Comment (Lars-DHondt-KUL) [inline DefaultSettings/get_git_hash.m:1]: Maybe add in the header that the git hash is used to identify a specific version of the code.

Comment (VDB-Bram) [reply DefaultSettings/get_git_hash.m:1]: header updated

File: DefaultSettings/get_git_hash.m [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+%
+% Last edit by: /
+% Last edit date: DD/MM/YYYY
+% --------------------------------------------------------------------------
+
+command = [ 'git rev-parse HEAD' ];
+[status,hash] = system(command);
+
+if( status ~= 0 )
+    error('Unable to get hash from file.');
```
[the remark this answers is not in the document]
Comment (VDB-Bram) [reply DefaultSettings/get_git_hash.m]: warning added

**Review by Lars-DHondt-KUL — DISMISSED** [review]

**Comment by dgupta7** [comment]
It might be useful to check if we can also check that if the hash exists, does it correspond to the latest version.
In other words, tell the user if they are using the latest version of the code on github or not.

**Comment by VDB-Bram** [comment]
> It might be useful to check if we can also check that if the hash exists, does it correspond to the latest version. In other words, tell the user if they are using the latest version of the code on github or not.

added

**Review by dgupta7 — APPROVED** [review]
It looks good, but I think that most users might not know what git hash means. So in the readme file where you describe these settings, maybe you can use a simpler language or briefly say what git hash means. :)

**Comment by VDB-Bram** [comment]
> It looks good, but I think that most users might not know what git hash means. So in the readme file where you describe these settings, maybe you can use a simpler language or briefly say what git hash means. :)

Fixed I think; clarification added to what is means.

**Pull request merged**